src/Terminus.php

```diff
@@ -163,8 +164,10 @@ private function addPluginsCommandsAndHooks()
         $discovery = $this->getContainer()->get(PluginDiscovery::class, [$this->getConfig()->get('plugins_dir'),]);
```

Comment:
Here's another two-parameter `$container->get()`. PluginDiscovery wouldn't have to be stored in the container; it could be instantiated once, here, since this is the only place it is used.